endpoint/OBDComp/emulator/script.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+SUDO=false
+if which sudo &>/dev/null; then
+	SUDO=true
+fi
+
+# create vcan
```

Comment:
If there is no `vcan`, you have to alert.

Reply:
That is, `vcan?` is unable to be created once the kernel lacks of `CONFIG_CAN_VCAN`.